identity_jose/Cargo.toml

```diff
@@ -21,9 +21,20 @@ thiserror.workspace = true
 url = { version = "2.3.1", default-features = false, features = ["serde"] }
 zeroize = { version = "1.5.7", default-features = false, features = ["std", "zeroize_derive"] }
 
+[features]
+# Enables static method `verify_eddsa_jws_prechecked_alg`
+eddsa = ["iota-crypto/ed25519"]
+# Enables construction of the default JWS `Decoder`.
+default-jws-signature-verifier = ["eddsa"]
```

Comment:
Can we not make a dedicated `EdDSAJwsSignatureVerifier` and gate that behind the `eddsa` feature. Future default implementations could add their own feature, so we wouldn't need `default-jws-signature-verifier`.